src/common/OptionButton.js

```diff
@@ -8,7 +8,7 @@ import { IconRight } from '../common/Icons';
 const styles = StyleSheet.create({
   optionRow: {
```

Comment:
Move this too `theme.js`
It is also repeated in `OptionRow.js`
In a another commit. (other than commit which adds icons)

Reply:
@jainkuniya Moved `optionRow` to `theme.js` as instructed.